spinalcordtoolbox/labels.py

```diff
@@ -464,3 +465,69 @@ def remove_other_labels_from_image(img: Image, labels: Sequence[int]) -> Image:
             logger.warning(f"Label {label} not found in input image!")
 
     return out
+
+
+def project_discs(img: Image, ref: Image, num_disc=True) -> Image:
+    """
+    Project discs coordinates on the spinal cord centerline. This projection is obtained by iterating along
+    the centerline to identify the shortest distance with each referenced coordinates.
+    Typically, user inputs a segmentation image, and labels with disks position, and this function computes
+    the identification to the closest coordinates of each labels on the centerline.
+    Labels are assumed to be non-zero and incremented from top to bottom
```

Comment:
I don't think you actually assume that the labels increment from top to bottom.

Reply:
I just forgot to erase this line when copying. See https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4080/commits/59e758e3e04d1217ce52ab8f24645776ef316214